L/libLLVM_jll/Compat.toml

```diff
@@ -1,5 +1,8 @@
 [8]
-julia = "1"
+julia = "1.4"
 
-[9-10]
-julia = "1.5"
+[9]
+julia = "1.5-1.6"
```

Comment:
Without this, LLVM 10 gets installed. that's incompatible, so I had it require Julia 1.7 for now.